### PR TITLE
feat: redesign the learn page and implements the explore page

### LINKED ISSUE #8842: A minor redesign of the web.dev homepage

### PR DESCRIPTION
**Pull request by mamieorine**
Fixes #8842

Changes proposed in this pull request:

- Redesign of the web.dev **learn** page (https://web.dev/learn/)
- Implements the web.dev **explore** page (/explore)

### CODE REVIEW

**Comment by mamieorine** [comment]
@devnook 

I have done redesign of the learn page. Please review: 
- https://deploy-preview-9232--web-dev-staging.netlify.app/learn/

Also, implemented a new explore page. Please review: 
- https://deploy-preview-9232--web-dev-staging.netlify.app/explore/

Figma: https://www.figma.com/file/eXb7xAA7XROYOYNHkHcMJg/%5Bweb.dev%5D-Master?node-id=481%3A18737&t=pE3KlHQApVgmp2o6-0

I used this PR for Learn and Explore pages because the new redesign of both pages is based on the old design of the learn page (https://web.dev/learn/). In addition, I did not move the course collections from _/learn_ to _/explore_ because I can see too many files changed over 400 files, so I decide to import all courses to explore page instead.

**Review by rachelandrew — APPROVED** [review]
This LGTM from a content perspective, though we don't want to merge it until we have done the navigation updates as there's currently no way to get to /explore.

File: src/site/content/pl/explore/index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+layout: explore
+title: Badać
```
Comment (devnook) [inline src/site/content/pl/explore/index.md:3]: Please replace with "Eksploruj"

**Comment by devnook** [comment]
I think courses should stay in /learn anyways, based on figma?

@mamieorine generally LGTM, just please add "Explore" to the top navigation as part of this PR (see figma).

File: src/site/content/pl/explore/index.md (hunk 1 above)
Comment (mamieorine) [reply src/site/content/pl/explore/index.md]: I have corrected the word for the pl language

**Comment by mamieorine** [comment]
@devnook  I have added an explore page into the nav bar on the header

**Comment by devnook** [comment]
The cards grid gets squashed (loses the gap) at circa ~ 1200px
Screenshot 2022-12-20 at 13 30 13

**Review by devnook — CHANGES_REQUESTED** [review]
Please check grid settings at 1288px

**Comment by rachelandrew** [comment]
It also doesn't seem to be responsive, it's overflowing at narrow widths. We should use the same grid as on the blog I think https://web.dev/blog/

**Comment by mamieorine** [comment]
@devnook @rachelandrew  I have fixed responsive broken on tablet and mobile views.

**Comment by rachelandrew** [comment]
@mamieorine that's looking a lot better thanks!

**Review by devnook — APPROVED** [review]

**Pull request merged**